### DISCUSSION TITLE
Crash reports sent to apple.

### DISCUSSION THREAD

**Discussion by origamifreak2** [Sequel-Ace/Sequel-Ace#1840] [Q&A] [answered]
Are the reports that show when an app crashes sent to the developers at all or just to apple? On a related note, would it be better for Sequel-Ace to send crash reports to apple and post them in github, or is posting in github sufficient?
![crash_report](https://github.com/Sequel-Ace/Sequel-Ace/assets/2578041/1e4c9b9d-b5f1-484b-88ac-51dab4c45980)

**Comment by Jason-Morcos** [comment, marked as answer]
General sent to Apple and to Sequel Ace's release managers via App Store Connect. But they're not super helpful in general.
Realistically, we don't have the bandwidth to investigate most one-off crashes. If you're able to develop and test locally to provide fixes, we're happy to review those. Otherwise, we only have bandwidth to look into crashes affecting large portions of the community.
